assignment-01.Rmd

```diff
@@ -9,11 +9,11 @@ output:
 knitr::opts_chunk$set(eval = FALSE)
 ```
 
-*To submit this assignment, upload the full document on blackboard,
+*To submit this assignment, upload the full document on Quercus,
 including the original questions, your code, and the output. Submit
-you assignment as a knitted `.pdf` (prefered) or `.html` file.*
+your assignment as a knitted `.pdf` (prefered) or `.html` file.*
 
-1.  Get setup at home (or on a lab computer after hours). (1.5 marks)
+1.  Get set up at home (or on a lab computer after hours). (1.5 marks)
```

Comment:
What about having them print the output of `sessionInfo()` or some equivalent? Basically to confirm that everyone is using the most recent version of R, which they should be.